src/views/routing/sidebar/body/SidebarBody.native.js

```diff
@@ -31,14 +33,23 @@ class Sidebar extends Component {
       headerLayout,
       footerLayout,
       renderBody,
+      sortDirection,
+      sortBy,
     } = this.props;
 
+    const sortedItems = [...items];
+
+    /* If both a sort direction and sort by are provided then sort the items */
+    if ( sortDirection && sortBy ) {
+      sortDirection === 'desc' ? sortedItems.sort(( a, b ) => b[sortBy] > a[sortBy] ? 1 : -1 ) : sortedItems.sort(( a, b ) => a[sortBy] > b[sortBy] ? 1 : -1 );
```

Comment:
try to clean this up a bit, most of the code is being duplicated. consider defining 'a' and 'b', and making a function that can be called in both cases ie:
const itemA = a[sortBy]
const itemB = b[sortBy]
const sortFunc = (a, b) => a > b ? 1 : -1;
sortedItems.sort(( a,b ) => sortDirection === 'asc' ? sortFunc(a, b) : sortFunc(b, a);